common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2366,7 +2366,7 @@ public static enum ConfVars {
         "Whether Hive enables the optimization about converting common join into mapjoin based on the input file size. \n" +
         "If this parameter is on, and the sum of size for n-1 of the tables/partitions for a n-way join is smaller than the\n" +
         "specified size, the join is directly converted to a mapjoin (there is no conditional task)."),
-    HIVE_CONVERT_ANTI_JOIN("hive.auto.convert.anti.join", true,
+    HIVE_CONVERT_ANTI_JOIN("hive.auto.convert.anti.join", false,
         "Whether Hive enables the optimization about converting join with null filter to anti join"),
```

Comment:
I would also add a small warning in the description that turning it to `true` may in some cases lead to wrong results or that at this stage it is an experimental feature or something along these lines. Without doing so users do not know if they should touch the configs or not.